app/assets/javascripts/darkswarm/controllers/cart_form_controller.js.coffee

```diff
@@ -0,0 +1,2 @@
+Darkswarm.controller "CartFormCtrl", ($scope) ->
+
```

Comment:
ah, this bootstraps angular basics on the form? I didnt know this :ok_hand:

Reply:
I think if there's no controller on an element, there's no `$scope` to play with. Initially I thought I would need to add some more code to this controller over the course of the PR, but in the end there was nothing else needed.